docs/recipes/stencil.md

```diff
@@ -0,0 +1,85 @@
+# Usage with Stencil
+
+[Stencil](https://stenciljs.com/) web components work very well with XState.
+
+### components/toggle/toggle-machine.ts

```

Comment:
I'd not place this alongside the component, unless the state machine is really just relevant for this single component?

```suggestion
### `src/helpers/toggle-machine.ts`
```